lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -157,14 +157,25 @@ SWAP_BP_FOR_OPCODE:
                 return nullptr;
 
             case INTERPRETER_OPCODE::ExtendedOpcodePrefix:
+            case INTERPRETER_OPCODE::DblExtendedOpcodePrefix:
```

Comment:
Ah yeah, that is true. I agree with you, having this behind #ifdef is good idea.